web/src/features/panels/zone/util.ts

```diff
@@ -77,3 +77,20 @@ export function getDisclaimer(zoneId: string) {
   const zoneConfig = config.zones[zoneId];
   return zoneConfig?.disclaimer;
 }
+
+export function showEstimationFeedbackCard(
```

Comment:
It would be nice to use the same pattern we use for the onboarding modal here and use an [atom with storage](https://jotai.org/docs/utilities/storage) it's stored as onboardingSeen. It could make sense to call it estimationFeedbackSeen and default it to false. The atoms are nice because you can read and set them easily from any component

Reply:
Good point! I've changed it in the commit [change to use atoms](https://github.com/electricitymaps/electricitymaps-contrib/pull/6464/commits/fee2b04bb2a1c9970e4ae0ff40445c9296bb3724)